### ISSUE TITLE
repeat to call rpc function cause "fatal error: unexpected signal during runtime execution"

### ISSUE THREAD

**Issue by wethura** [frida/frida-go#18]
## Problem

while i call the rpc function repeat many times, it will crash: "fatal error: unexpected signal during runtime execution"

## How to reproduce: 

call function in your for loop, may be you can get the same problem.

## Env

- system: Linux 6.1.31-2-MANJARO SMP PREEMPT_DYNAMIC Sun Jun  4 12:31:46 UTC 2023 x86_64 GNU/Linux
- golang: go version go1.20.4 linux/amd64
- emulator: android studio
- avd: Android 13 Google APIs | x86_64

## Code

```go
const scriptAdd = `
rpc.exports = {
    add: function (a, b) { return a + b; },
};
`

func TestFrida(t *testing.T) {
	emulator, err := frida.USBDevice().Attach("鲁大师", nil)
	require.NoError(t, err)

	createScript, err := emulator.CreateScript(scriptAdd)
	require.NoError(t, err)

	createScript.On("message", func(msg string) {
		//fmt.Println("[*] Received", msg)
	})

	err = createScript.Load()
	require.NoError(t, err)

	for i := 0; i < 10000; i++ {
		createScript.ExportsCall("add", 1, 2)
	}
}
```

## Error

```txt
fatal error: unexpected signal during runtime execution
[signal SIGSEGV: segmentation violation code=0x80 addr=0x0 pc=0x8f2db9]

runtime stack:
runtime.throw({0x2b91d44?, 0x307c0?})
	/usr/lib/go/src/runtime/panic.go:1047 +0x5d fp=0x7f817b7fdd30 sp=0x7f817b7fdd00 pc=0x57311d
runtime.sigpanic()
	/usr/lib/go/src/runtime/signal_unix.go:821 +0x3e9 fp=0x7f817b7fdd90 sp=0x7f817b7fdd30 pc=0x589d29
```

if you run the upper code and the error can not be reproduce, rerun the code immediately.

**Comment by NSEcho** [comment]
Hey there, thank you for detailed report. I believe I have solved the issue, can you please try using [v0.6.6](https://github.com/frida/frida-go/releases/tag/v0.6.6) and see if it is solved now, I have tested it with 10k calls and it did not fail.

**Comment by wethura** [comment]
> Hey there, thank you for detailed report. I believe I have solved the issue, can you please try using [v0.6.6](https://github.com/frida/frida-go/releases/tag/v0.6.6) and see if it is solved now, I have tested it with 10k calls and it did not fail.

Yes, the err no found in v0.6.6
